test/nn/conv/BayesianGCNConv_test.py

```diff
@@ -0,0 +1,128 @@
+import copy
+
+import torch
+from torch_sparse import SparseTensor
+
+from torch_geometric.nn.conv.BayesianGCNConv import BayesianGCNConv
```

Comment:
```suggestion
from torch_geometric.nn import BayesianGCNConv
```
You might need to add `BayesianGCNConv` into `__init__.py` in `conv/`.